src/annotator/integrations/html.ts

```diff
@@ -32,12 +37,14 @@ export class HTMLIntegration extends TinyEmitter implements Integration {
   container: HTMLElement;
   features: FeatureFlags;
 
-  private _flagsChanged: () => void;
+  private _flagsChanged?: () => void;
```

Comment:
You shouldn't need the changes to `_flagsChanged`. When the flag changes this code calls `fitSideBySide` in the same way as the `Guest` does. If `fitSideBySide` handles the case where side-by-side mode is manual, then it should apply whether the Guest is calling that method or the flags-changed handler is.

Also you should be aware that the feature flags are typically not loaded when this class is instantiated. Instead they are loaded later on after the sidebar app has loaded.